/*
 * This file was automatically generated by EvoSuite
 */

package net.sf.xisemele.impl;

import org.junit.Test;
import org.junit.runner.RunWith;
import org.evosuite.junit.EvoSuiteRunner;
import static org.junit.Assert.*;
import net.sf.xisemele.impl.NullErrorHandler;
import org.xml.sax.Locator;
import org.xml.sax.SAXException;
import org.xml.sax.SAXParseException;
import org.xml.sax.ext.Locator2Impl;

@RunWith(EvoSuiteRunner.class)
public class NullErrorHandlerEvoSuiteTest {


  //Test case number: 0
  /*
   * 2 covered goals:
   * 1 net.sf.xisemele.impl.NullErrorHandler.warning(Lorg/xml/sax/SAXParseException;)V: root-Branch
   * 2 net.sf.xisemele.impl.NullErrorHandler.<init>()V: root-Branch
   */
  @Test
  public void test0()  throws Throwable  {
      NullErrorHandler nullErrorHandler0 = new NullErrorHandler();
      Locator2Impl locator2Impl0 = new Locator2Impl();
      SAXParseException sAXParseException0 = new SAXParseException("", (Locator) locator2Impl0);
      nullErrorHandler0.warning(sAXParseException0);
  }

  //Test case number: 1
  /*
   * 1 covered goal:
   * 1 net.sf.xisemele.impl.NullErrorHandler.error(Lorg/xml/sax/SAXParseException;)V: root-Branch
   */
  @Test
  public void test1()  throws Throwable  {
      NullErrorHandler nullErrorHandler0 = new NullErrorHandler();
      nullErrorHandler0.error((SAXParseException) null);
  }

  //Test case number: 2
  /*
   * 2 covered goals:
   * 1 net.sf.xisemele.impl.NullErrorHandler.fatalError(Lorg/xml/sax/SAXParseException;)V: root-Branch
   * 2 net.sf.xisemele.impl.NullErrorHandler.<init>()V: root-Branch
   */
  @Test
  public void test2()  throws Throwable  {
      NullErrorHandler nullErrorHandler0 = new NullErrorHandler();
      nullErrorHandler0.fatalError((SAXParseException) null);
  }
}
